test_xdb/models/under_test/split_test.sql

```diff
@@ -1,12 +1,14 @@
 /* TODO split is supported for bigquery. tests should support that */
```

Comment:
Since this fixes it, might as well remove the `TODO` comment along with these changes.

Reply:
whoops, just pushed up a commit to remove it!